neurons/remote_config.py

```diff
@@ -65,7 +65,7 @@ def load_remote_config(self):
                     break  # Break the loop if successful
                 except requests.exceptions.RequestException as e:
                     retries += 1
-                    bt.logging.error("Attempt failed to update config from", retries = retries, config_url = self.config_url, error = {'exception_type': e.__class__.__name__,'exception_message': str(e),'exception_args': e.args})
+                    bt.logging.error("Attempt failed to update config", retries = retries, config_url = self.config_url, error = {'exception_type': e.__class__.__name__,'exception_message': str(e),'exception_args': e.args})
```

Comment:
for some reason if reties as int is thorws serialization error, my work around was to str(retires),m pls double check